microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,88 @@
+
+The default recommended value is “limited.” With limited, the above risks are mitigated by ensuring that “incompatible” combinations of language and region are not revealed to websites. The “limited” value ensures that only “valid” combinations of the user’s current browser display language preference and a given region value are used. This ensures that websites won’t see a “regional code” anomaly (though it could still look unusual per geographic inference), and also ensures that for language processing, websites will at least provide a consistent language experience (not mixing languages in weird combinations). The “limited” setting compares the language part of the regional code that is derived from the browser’s display language, and the regional part coming from the OS setting, and if the language parts match and if the resulting regional code is a valid combination, will pass the result through to the website. If the language parts don’t match or if the resultant combination isn’t valid, then the OS region preference is silently ignored and the feature acts as if “never” was selected (the regional information falls back to being derived from the browser’s display language). 
+
+Note: the “limited” value is similar to current Firefox behavior for exposing the regional code as described in [Locale management — Firefox Source Docs documentation (mozilla.org)](https://firefox-source-docs.mozilla.org/intl/locale.html#regional-preferences), but also offers the possibility to not share this information at all. 
+
+## Policy Controls 
+A related new policy added to Microsoft Edge provides a setting for corporate or enterprise-level default configurations: **ShareOSRegionThroughJavaScriptLocale** which accepts an integer with potential values: 
+
+* 0 = Limited 
+* 1 = Always 
+* 2 = Never 
+
+In addition, an [older policy](https://docs.microsoft.com/en-us/DeployEdge/microsoft-edge-policies#applicationlocalevalue) (**ApplicationLocaleValue**) allows setting the Edge locale and prevents users from changing it. 
+
+## Developer Details 
+The new setting to “Share additional OS regional format” directly impacts the JavaScript `Intl` object’s default regional value as well as all other JavaScript APIs that use the default region (e.g., such as the `Date` constructor). In cases where the `Intl` object’s constructors are used to create a new `Region`, `DateTimeFormat`, etc., with an existing regional code, these behaviors are unchanged. However, the Constructors offer options to create the objects without a specified regional code, in which case, the APIs fallback to a default regional code. The default regional code is what is surfaced via the new user preference. For example, a `DateTimeFormat` can be created with the user’s default regional code (and inspected) via: 
```

Comment:
i'll change eg and etc but probably keep via

Reply:
edited section